fastlane/lib/fastlane/actions/spm.rb

```diff
@@ -93,7 +102,15 @@ def self.available_options
                                        env_name: "FL_SPM_VERBOSE",
                                        description: "Increase verbosity of informational output",
                                        type: Boolean,
-                                       default_value: false)
+                                       default_value: false),
+          FastlaneCore::ConfigItem.new(key: :simulator,
+                                       env_name: "FL_SPM_SIMULATOR",
+                                       description: "Specifies the simulator to pass for Swift Compiler",
+                                       type: String,
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Please pass a valid simulator: (iphonesimulator|macossimulator)") unless valid_simulators.include?(value)
```

Comment:
nitpick: we could reuse the list here

```
UI.user_error!("Please pass a valid simulator: (#{self.valid_simulators.join('|')})") unless valid_simulators.include?(value)
```

Looks good otherwise.

Reply:
Done